jobserver/api/jobs.py

```diff
@@ -118,6 +118,8 @@ def post(self, request, *args, **kwargs):
                     )
 
                     span.set_attribute("job_id", str(job.id))
+                    span.set_attribute("identifier", job.identifier)
```

Comment:
I would call this "job_id"

Reply:
willing to go to `job_identifier`? lots of potential confusion between id / identifier in this class...